pages/rdm_cycle/re-using.md

```diff
@@ -1,8 +1,48 @@
 ---
 title: Re-using
 keywords:
+contributors: [korbinib, DanFaria]
 ---
 
-## Subtitle 1
+## What is Data Re-use?
 
-Some text that you can fill in
\ No newline at end of file
+Data re-use describes the process of using available data for own, new research.
+
+## Why Data Re-use is important?
+
+By re-using existing data you can:
+
+* obtain reference data for your research
+* avoid doing new, unecessary experiments
+* make research more robust by aggregating results obtained from different methods or samples
+* gain novel insights by connecting and meta-analysing data-sets
+
+## What should be considered for Data Re-use?
+
+One major challenge is to find data for re-use. Another point is that you should assess the quality of the deposited data before it is re-used.
+One starting point can be to use value added databases with curatated content. Other possibities include searching data deposition repositories for suitable datasets based on their annotation or obtaining data directly from the author. You should keep in mind whether the repositories provide sufficient metadata to enable data re-use. Some types of data you are searching for can be straightforward to re-use (e.g. genome data) or require extensive metadata to interpret and reuse (e.g. gene expression experiment data)
+The re-use of data should be aligned with the intellectual property right e.g. by making sure it is available under a suiting license.
```

Comment:
Correct suiting license to "suitable license"

Reply:
Thanks!